build.gradle.kts

```diff
@@ -31,18 +31,17 @@ subprojects {
 
     spotless {
         val exclude = listOf(
-            "**/.github/**",
-            "**/.gradle/**",
-            "**/.idea/**",
-            "**/.intellijPlatform/**",
-            "**/.kotlin/**",
+            ".github/**",
```

Comment:
Why change this? I thought that with the "**/" it would exclude also from subfolders.